### PR TITLE
[EN/ZH] Tournament-CJB: Finals - Week 2

### PR DESCRIPTION
**Pull request by kj415j45**

### CODE REVIEW

File: wiki/Tournaments/CJB/2018/4K/en.md [hunk 1]
```diff
@@ -174,11 +162,22 @@ If *Crystal* **lose** Match 30, Match 31 will start right after that: *Crystal*
 
 ## Match Results
 
+### Finals - Week 2
+
+|  No. | Monday, 2018-08-13 |  |  |  |  |
```
Comment (fajar13k) [inline wiki/Tournaments/CJB/2018/4K/en.md]: Just need 1 Space before No.

File: wiki/Tournaments/CJB/2018/4K/en.md [hunk 2]
```diff
@@ -46,23 +46,11 @@ Prizes for this tournament are donated by ![][flag_CN] [hans1999](https://osu.pp
...
+--------------------------------------------------
 
-### Finals - Week 2
-
-#### Monday, 13. August 2018
-
-| Match Time | Player 1 |  |  | Player 2 | Match ID |
-| :-: | --: | --: | :-- | :-- | :-: |
-| 20:00 | Skyshifter | ![][flag_CN] | ![][flag_CN] | Toki 0w0 | 29 |
+![CJB 4K 4th podium](podium.jpg)
```
Comment (fajar13k) [inline wiki/Tournaments/CJB/2018/4K/en.md]: Place all images to a subfolder named `img`?

**Comment by fajar13k** [comment]
Also, if the tournament is already concluded. You may want to write the tournament to the Tournaments index page.

File: wiki/Tournaments/CJB/2018/4K/en.md [hunk 3]
```diff
@@ -1,16 +1,22 @@
+---
+tags:
+- CJB
+- CJB 2018
+- CJB2018
+---
 The 4th CJB osu!mania 4K Tournament
```
Comment (Joehuu) [inline wiki/Tournaments/CJB/2018/4K/en.md]: Change the header format to `#`.

File: wiki/Tournaments/CJB/2018/4K/en.md [hunk 4]
```diff
@@ -1,16 +1,22 @@
...
 =========================
 
-![CJB 4K 4th](banner.png)
+![CJB 4K 4th](img/banner.png)
 
-**The 4th CJB osu!mania 4K Tournament _(CJB 4K 4th)_**, is the fourth one-on-one osu!mania 4K specified Chinese regional tournament hosted by CJB team.
+**The 4th CJB osu!mania 4K Tournament _(CJB 4K 4th)_**, is a one-on-one osu!mania 4K specified Chinese regional tournament hosted by the CJB team. It is the 4th installment of the CJB osu!mania 4K Tournament.
```
Comment (Joehuu) [inline wiki/Tournaments/CJB/2018/4K/en.md]: `**The 4th CJB osu!mania 4K Tournament _(CJB 4K 4th)_**` -> 
`**The 4th CJB osu!mania 4K Tournament** (***CJB 4K 4th***)`

Remove the comma before `is`.

File: wiki/Tournaments/CJB/2018/4K/en.md [hunk 5]
```diff
@@ -19,15 +25,15 @@ The 4th CJB osu!mania 4K Tournament
 
 ## Prizes
 
-Prizes for this tournament are donated by ![][flag_CN] [hans1999](https://osu.ppy.sh/users/6679329).
+Prizes for this tournament are sponsored by ![][flag_CN] [hans1999](https://osu.ppy.sh/users/6679329).
 
 | Placing | Prize(s) |
 | --- | :-- |
```
Comment (Joehuu) [inline wiki/Tournaments/CJB/2018/4K/en.md]: Center crowns.

File: wiki/Tournaments/CJB/2018/4K/en.md [hunk 6]
```diff
@@ -46,23 +52,11 @@ Prizes for this tournament are donated by ![][flag_CN] [hans1999](https://osu.pp
...
+--------------------------------------------------
 
-### Finals - Week 2
-
-#### Monday, 13. August 2018
-
-| Match Time | Player 1 |  |  | Player 2 | Match ID |
-| :-: | --: | --: | :-- | :-- | :-: |
-| 20:00 | Skyshifter | ![][flag_CN] | ![][flag_CN] | Toki 0w0 | 29 |
+![CJB 4K 4th podium](img/podium.jpg)
```
Comment (Joehuu) [inline wiki/Tournaments/CJB/2018/4K/en.md]: `CJB 4K 4th podium` -> `CJB 4K 4th Podium`

File: wiki/Tournaments/CJB/2018/4K/en.md [hunk 7]
```diff
@@ -214,7 +219,7 @@ If *Crystal* **lose** Match 30, Match 31 will start right after that: *Crystal*
 | 10 | **tuziel** ![][flag_CN] | **5** | 0 | ![][flag_CN] Snowkyou | |
 | 14 | IA daisuki ![][flag_CN] | 0 | **5** | ![][flag_CN] **Toki 0w0** | |
 
-Note: ![][flag_CN] Jazzsoldiers give up tournament for some reasons.
+_Note: ![][flag_CN] Jazzsoldiers gave up tournament for some reasons._
```
Comment (Joehuu) [inline wiki/Tournaments/CJB/2018/4K/en.md]: Use aterisks(`*`) for every italicize and bold.

**Pull request merged**